### DISCUSSION TITLE
How to open a file?

### DISCUSSION THREAD

**Discussion by ssiyad** [nvim-neorg/neorg#1282] [Q&A] [answered]
For example, I have a todo file at `~/neorg/todo.norg`. I want to open this file using either with `:Neorg` or the lua api. How can I achieve this?

**Comment by katawful** [comment, marked as answer]
`vim.cmd.edit("file")` in lua

`:edit file` in command-line mode

**Reply by ssiyad** [reply]
Thanks!
